cogboard-webapp/src/actions/thunks.js

```diff
@@ -126,6 +128,64 @@ const reorderWidgetsThunk = (sourceId, targetIndex) => (dispatch, getState) => {
   dispatch(sortWidgets({ sourceId, targetIndex, boardId }));
 };
 
+const loadSettingsThunk = token => dispatch => {
+  const init = {
+    headers: {
+      Authorization: token ? token : getToken()
```

Comment:
`token || getToken()`?

Reply:
Ups. It was for test purpose, I should remove it. Fixed. ;)